cmd/main.go

```diff
@@ -218,6 +230,13 @@ func main() {
 		os.Exit(1)
 	}
 
+	if enableWebhook {
+		if err := mgr.Add(webhookServer); err != nil {
+			os.Exit(1)
+		}
+
+		webhookServer.Register("/validate-plan", w.ValidateWebhook(mgr.GetClient()))
```

Comment:
Going to also suggest we use a more descriptive name than "w" for those reading.